config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
+  resources :dishes
   devise_for :users
-  root to: 'pages#home'
+  get '/menu_days', to: 'menus#index'
+  get '/today', to: 'menus#select_day'
+  root to: 'dishes#index'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
```

Comment:
1. devise on top
2. Blank line after devise
3. instead of get '' use resources :menu_day (singular), only: %i(show)